Assignments/Program.cs

```diff
@@ -1,3 +1,42 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T27 - Implementing Stack<T> class
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+namespace Spark23;
+
+public class T27 {
+   public static void Main () {
+      var stack = new TStack<int> ();
+      stack.Push (1);
+      stack.Peek ();
+      stack.Pop ();
+   }
+   public class TStack<T> {
+      public bool IsEmpty => mIndex == 0;
+      public int Size => mIndex;

```

Comment:
Is is Index or Count? Name of the property not clear.
mIndex is zero based, Size or Count will be one more than index right?
Since you are incrementing it matches the count but the same variable can't be used for different purpose.